open-metadata-test/open-metadata-fvt/access-services-fvt/data-engine-fvt/src/test/java/org/odpi/openmetadata/accessservices/dataengine/fvt/DataEngineFVT.java

```diff
@@ -0,0 +1,94 @@
+import org.odpi.openmetadata.repositoryservices.connectors.stores.metadatacollectionstore.properties.instances.Relationship;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.fail;
+
+public class DataEngineFVT {
+
+    private static final String DESCRIPTION = "description";
+    private static final String NAME = "name";
+    private static final String TYPE = "type";
+    private static final String VERSION = "version";
+    private static final String PATCH_LEVEL = "patchLevel";
+    private static final String QUALIFIED_NAME = "qualifiedName";
+    private static final String SOURCE = "source";
+
+    private final ProcessSetupService processSetupService = new ProcessSetupService();
+
+    @ParameterizedTest
+    @MethodSource("org.odpi.openmetadata.accessservices.dataengine.PlatformConnectionProvider#getConnectionDetails")
+    public void registerExternalTool(String userId, DataEngineClient dataEngineClient, RepositoryService repositoryService)
+            throws InvalidParameterException, UserNotAuthorizedException, PropertyServerException, ConnectorCheckedException {
+
+        SoftwareServerCapability softwareServerCapability = processSetupService.createExternalDataEngine(userId, dataEngineClient);
+
+        Optional<EntityDetail> entityDetail = repositoryService.findSoftwareServerCapabilityByPropertyValue(softwareServerCapability.getQualifiedName());
+
+        if(!entityDetail.isPresent()) {
+            fail();
+        }
+
+        EntityDetail entity = entityDetail.get();
+        assertEquals(softwareServerCapability.getDescription(), entity.getProperties().getPropertyValue(DESCRIPTION).valueAsString());
+        assertEquals(softwareServerCapability.getDisplayName(), entity.getProperties().getPropertyValue(NAME).valueAsString());
+        assertEquals(softwareServerCapability.getEngineType(), entity.getProperties().getPropertyValue(TYPE).valueAsString());
+        assertEquals(softwareServerCapability.getEngineVersion(), entity.getProperties().getPropertyValue(VERSION).valueAsString());
+        assertEquals(softwareServerCapability.getPatchLevel(), entity.getProperties().getPropertyValue(PATCH_LEVEL).valueAsString());
+        assertEquals(softwareServerCapability.getQualifiedName(), entity.getProperties().getPropertyValue(QUALIFIED_NAME).valueAsString());
+        assertEquals(softwareServerCapability.getSource(), entity.getProperties().getPropertyValue(SOURCE).valueAsString());
+
+    }
+
+    @ParameterizedTest
+    @MethodSource("org.odpi.openmetadata.accessservices.dataengine.PlatformConnectionProvider#getConnectionDetails")
+    public void verifyLineageMappingsForAJobProcess(String userId, DataEngineClient dataEngineClient, RepositoryService repositoryService)
```

Comment:
I am missing something here, how does the lineage graph get populated for this FVT?